exp-player/addon/components/exp-lookit-story-page/component.js

```diff
@@ -336,6 +347,16 @@ export default ExpFrameBaseUnsafeComponent.extend(FullScreen, VideoRecord,  {
         }
     },
 
+    parentTextStyle: Ember.computed('parentTextBlock', function() {
+        var parentTextBlock = this.get('parentTextBlock') || {};
+        var css = parentTextBlock.css || {};
+        var cssString = '';
+        $.each(css, function(propName, value) {
+            cssString += new Ember.Handlebars.SafeString(`${propName}:  ${value};`);
```

Comment:
I think that would be ok, or at least a big improvement. That said: the more you let the user write code directly in the page, the more risk there is of something going weird in general. Long term you may want to consider developing user stories for a very limited set of style options. This would mitigate the security risks, and ensure the site had a less buggy, more consistent look and feel.

Note that jquery loves to give a method call multiple forms; it will also take your object directly. Might be useful.
> css( properties )
properties
Type: PlainObject
An object of property-value pairs to set.